From 363bbf3135a22635fc014aed22c33f55a041b096 Mon Sep 17 00:00:00 2001
From: Ethan Chen <intervigil@gmail.com>
Date: Sun, 24 Sep 2017 18:29:55 -0700
Subject: [PATCH 2/2] wcnss_service: Read serial number from custom property

* ro.serialno and ro.boot.serialno are labeled as serialno_prop, which
  is restricted in the domains which are allowed to read it.
* Add a duplicate serial number property for wcnss_service to read.

Change-Id: I0b0cd68c399d2e711788c8c1f91c81e7db2f06e9
---
 wcnss-service/wcnss_service.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wcnss-service/wcnss_service.c b/wcnss-service/wcnss_service.c
index f064fa1..eafa81b 100644
--- a/wcnss-service/wcnss_service.c
+++ b/wcnss-service/wcnss_service.c
@@ -417,7 +417,7 @@ void setup_wcnss_parameters(int *cal)
 		return;
 	}
 
-	rc = property_get("ro.serialno", serial, "");
+	rc = property_get("ro.wcnss.serialno", serial, "");
 	if (rc) {
 		serial_num = convert_string_to_hex(serial);
 		ALOGE("Serial Number is  %x", serial_num);
-- 
2.17.1

